droid-command-line/src/main/java/uk/gov/nationalarchives/droid/command/action/CommandFactoryImpl.java

```diff
@@ -153,6 +157,10 @@ public DroidCommand getExportFormatCommand(final CommandLine cli) throws Command
             cmd.setColumnsToWrite(columns);
         }
 
+        if (cli.hasOption(CommandLineParam.EXPORT_TEMPLATE.getLongName())) {
```

Comment:
They are largely duplicates. One is Export "per file" and the other is export "per format" 
I don't think this duplication is causing any problem. I understand you are coming from the DRY principle, however in this case I would prefer to leave them as they are, Droid has some quirks around groups of parameters being valid in one case but not in another etc. and I would prefer to avoid any unforeseen  accidents :)